openstudiocore/CMakeLists.txt

```diff
@@ -1366,6 +1371,9 @@ include_directories("${CMAKE_HOME_DIRECTORY}/src/")
 # For QWinWidget
 include_directories("${CMAKE_HOME_DIRECTORY}/src/qtwinmigrate")
 
+# XML
+include_directories("${CMAKE_HOME_DIRECTORY}/src/pugixml/src")
```

Comment:
This might be what triggered the huge rebuild

Reply:
I also added this which will eventually help, https://github.com/NREL/OpenStudio/pull/3373